resources/config.yml

```diff
@@ -24,4 +24,4 @@ detection:
 repeat: 1
 
 # Plugin's version, please don't touch it !
-version: "2.0.0"
\ No newline at end of file
+version: "2.0.1"
```

Comment:
2.1.0 for a change that breaks compatibility

Reply:
Fixed: https://github.com/Symphonia-Team/ChestFinder/pull/16/commits/53938e81886affe447be2a5340182f975be9f44f